src/main/java/eu/dissco/core/translator/terms/BiocaseDigitalObjectDirector.java

```diff
@@ -0,0 +1,101 @@
+package eu.dissco.core.translator.terms;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import eu.dissco.core.translator.Profiles;
+import eu.dissco.core.translator.component.RorComponent;
+import eu.dissco.core.translator.properties.FdoProperties;
+import eu.dissco.core.translator.properties.WebClientProperties;
+import eu.dissco.core.translator.schema.Citations;
+import eu.dissco.core.translator.schema.DigitalSpecimen;
+import eu.dissco.core.translator.schema.Identifications;
+import eu.dissco.core.translator.terms.specimen.identification.AbcdTypeInformation;
+import java.util.ArrayList;
+import java.util.List;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.context.annotation.Profile;
+import org.springframework.stereotype.Component;
+
+@Slf4j
+@Component
+@Profile(Profiles.BIOCASE)
+public class BiocaseDigitalObjectDirector extends BaseDigitalObjectDirector {
+
+  public BiocaseDigitalObjectDirector(ObjectMapper mapper, TermMapper termMapper,
+      RorComponent rorComponent, WebClientProperties webClientProperties,
+      FdoProperties fdoProperties) {
+    super(mapper, termMapper, rorComponent, webClientProperties, fdoProperties, identifierTerms());
+  }
+
+  private static List<String> identifierTerms() {
+    var list = new ArrayList<String>();
+    list.add("abcd:id");
+    list.add("abcd:unitID");
+    list.add("abcd:unitIDNumeric");
+    list.add("abcd:unitGUID");
+    list.add("abcd:recordURI");
+    return list;
+  }
+
+  @Override
+  protected void addStandardSpecificLogic(DigitalSpecimen ds, JsonNode data, boolean dwc) {
```

Comment:
you don't use the dwc flag here (and can remove it from the parent function as well)